### ISSUE TITLE
another SLES 12 SP5 post-build check

### ISSUE THREAD

**Issue by czanik** [syslog-ng/syslog-ng#4359]
There was a similar problem a few weeks ago: https://github.com/syslog-ng/syslog-ng/issues/4331

Since last week, when I last built git snapshot on SLES 12 SP5, a similar problem was introduced, so package building fails. The error message while compiling 4.1.0 release is:
```
[  126s] ... testing for serious compiler warnings
[  126s]     (using /usr/lib/build/checks-data/check_gcc_output)
[  126s]     (using //.build.log)
[  126s] 
[  126s] I: Program is using implicit definitions of special functions.
[  126s]    these functions need to use their correct prototypes to allow
[  126s]    the lightweight buffer overflow checking to work.
[  126s]      - Implicit memory/string functions need #include <string.h>.
[  126s]      - Implicit *printf functions need #include <stdio.h>.
[  126s]      - Implicit *printf functions need #include <stdio.h>.
[  126s]      - Implicit *read* functions need #include <unistd.h>.
[  126s]      - Implicit *recv* functions need #include <sys/socket.h>.
[  126s] E: syslog-ng implicit-fortify-decl lib/stats/stats-control.c:49
```

**Comment by czanik** [comment]
@bazsi  Yes, it resolved the post build check problem for SLES 12 SP5 (and did not break other SUSE / openSUSE builds :-) ). Thanks!

**Comment by bazsi** [comment]
This was fixed in 4.1.1
